features/org.jaggeryjs.server.feature/pom.xml

```diff
@@ -25,10 +25,9 @@
     <modelVersion>4.0.0</modelVersion>
     <artifactId>org.jaggeryjs.server.feature</artifactId>
     <packaging>pom</packaging>
-    <name>Jaggery Server Feature</name>
+    <name>Jaggery Server Feature (without tools dependency)</name>
```

Comment:
we will use the original name and description

Reply:
https://github.com/wso2/jaggery/pull/213/commits/6795bbb5ffaa89c1cf9ad7b904fbc1148bfc73a7